shards/fast_string/fmt.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef C15A8B87_4A27_4D56_AD9E_4DA2AACA21FE
+#define C15A8B87_4A27_4D56_AD9E_4DA2AACA21FE
+
+#include "fast_string.hpp"
+#include <spdlog/fmt/fmt.h>
+
+// template<> struct fmt::formatter<shards::fast_string::FastString> {
```

Comment:
Apparently fmt somhow does this automatically already, probably using the implicit conversion, remove this.